client/dashboard/sites/logs/dataviews/style.scss

```diff
@@ -47,3 +47,16 @@
 		color: #2c3338;
 	}
 }
+
+/* Scroll to top button */
+.site-logs-card .site-logs-scroll-to-top {
+	position: absolute;
+	right: 5px;
+	bottom: 5px;
+	border: none;
+	border-radius: 50%;
+	box-shadow: 0 2px 6px rgba( 0, 0, 0, 0.3 );
+	color: var( --wp-admin-theme-color );
```

Comment:
I'm not sure if it's worth customizing the color 🤔

Reply:
Possibly not. You could argue that it stands out more as a link this way, but the same coloring shows on hover, and the arrow also helps indicate it's functionality. I'll remove it, can always update it in the future.